### PR TITLE
Allow users to modify their run message.

### PR DESCRIPTION
**Pull request by peregrineshahin**

### CODE REVIEW

**Comment by peregrineshahin** [comment]
Closes https://github.com/glinscott/fishtest/issues/1453

**Comment by ppigazzini** [comment]
DEV updated. The Info box is prefilled with 13 blank spaces, so the default message is blocked. After writing a new message and pushing "Submit" the test info doesn't change.

File: server/fishtest/views.py [hunk 1]
```diff
@@ -1005,6 +1007,12 @@ def tests_modify(request):
         run["args"]["priority"] = int(request.POST["priority"])
         run["args"]["throughput"] = int(request.POST["throughput"])
         run["args"]["auto_purge"] = True if request.POST.get("auto_purge") else False
+        if (
+            is_same_user(request, run)
+            and "msg-new" in request.POST
+            and request.POST["msg-new"].strip() != ""
+        ):
+            run["args"]["msg_new"] = request.POST["msg-new"].strip()
```
Comment (ppigazzini) [inline server/fishtest/views.py:1015]:
```diff
- run["args"]["msg_new"] = request.POST["msg-new"].strip()
+ run["args"]["info"] = request.POST["msg-new"].strip()
```

Comment (peregrineshahin) [reply server/fishtest/views.py:1015]: > ```diff
> - run["args"]["msg_new"] = request.POST["msg-new"].strip()
> + run["args"]["info"] = request.POST["msg-new"].strip()
> ```

not really try now at DEV

Comment (ppigazzini) [reply server/fishtest/views.py:1015]: Look at DEV, you are changing the wrong information.

**Comment by peregrineshahin** [comment]
Ah I'm changing the bench message instead of the info message thanks for showing me the run structure.

File: server/fishtest/views.py (hunk 1 above)
Comment (ppigazzini) [reply server/fishtest/views.py:1015]: "msg_base" and "msg_new" are the commit messages

**Comment by ppigazzini** [comment]
DEV updated.

**Review by ppigazzini — APPROVED** [review]
Tested writing HTML tags and UTF-8 characters

**Comment by ppigazzini** [comment]
PROD updated, thank you @peregrineshahin :)

**Pull request merged**